includes/Integrations/Wcfm/OrderMigrator.php

```diff
@@ -0,0 +1,416 @@
+                            }
+                        }
+                    }
+
+                    // Item Shipping Refund
+                    $vendor_shipping = $this->get_order_vendor_shipping();
+
+                    $shipping_tax  = 0;
+                    $shipping_cost = $shipping_tax;
+
+                    if ( ! empty( $vendor_shipping ) && isset( $vendor_shipping[ $vendor_id ] ) && $vendor_shipping[ $vendor_id ]['shipping_item_id'] ) {
+                        $shipping_item_id = $vendor_shipping[ $vendor_id ]['shipping_item_id'];
+                        $package_qty      = absint( $vendor_shipping[ $vendor_id ]['package_qty'] );
+
+                        ! $package_qty ? $package_qty = $line_item->get_quantity() : '';
+
+                        $shipping_item = new \WC_Order_Item_Shipping( $shipping_item_id );
+                        $refund_shipping_tax = $shipping_item->get_taxes();
+                        $shipping_tax_refund = array();
+
+                        if ( ! empty( $refund_shipping_tax ) && is_array( $refund_shipping_tax ) ) {
+                            if ( isset( $refund_shipping_tax['total'] ) ) {
+                                $refund_shipping_tax = $refund_shipping_tax['total'];
+                            }
+
+                            foreach ( $refund_shipping_tax as $refund_shipping_tax_id => $refund_shipping_tax_price ) {
+                                $refunded_amount += round( ( (float) $refund_shipping_tax_price / $package_qty ) * $line_item->get_quantity(), 2 );
+                                $shipping_tax_refund[ $refund_shipping_tax_id ] = round( ( (float) $refund_shipping_tax_price / $package_qty ) * $line_item->get_quantity(), 2 );
+                            }
+                        }
+
+                        $shipping_cost = (float) round( ( $vendor_shipping[ $vendor_id ]['shipping'] / $package_qty ) * $line_item->get_quantity(), 2 );
+                        $refunded_amount += $shipping_cost;
+                        $line_items[ $shipping_item_id ] = array(
+                            'refund_total' => $shipping_cost,
+                            'refund_tax'   => $shipping_tax_refund,
+                        );
+                    }
+                }
+
+                if ( ! empty( $refund_tax ) && is_array( $refund_tax ) ) {
+                    foreach ( $refund_tax as $refund_tax_id => $refund_tax_price ) {
+                        $refunded_amount += (float) $refund_tax_price;
+                    }
+                }
+
+                try {
+                    $line_items[ $item_id ] = array(
+                        'refund_total' => $c_refunded_amount,
+                        'refund_tax'   => $refund_tax,
+                    );
+
+                    $line_items[ $item_id ]['qty'] = $c_refunded_qty;
+
+                    $wcfm_create_refund_args = array(
+                        'amount'         => round( $refunded_amount, 2 ),
```

Comment:
Instead of using `round`, you could use `Automattic\WooCommerce\Utilities\NumberUtil::round` as recommended.